assembly/__tests__/runtime/roundtrip.spec.ts

```diff
@@ -80,8 +80,44 @@ describe("Round Trip", () => {
       arr.pushFront("hello");
       expect<PersistentDeque<string>>(roundtrip(arr)).toStrictEqual(arr);
     });
-  }); 
-})
+  });
+
+  describe("Map", () => {
+    beforeEach(() =>{
+      map = new Map<string, string>();
+    });
+
+    it("should handle an empty map", () => {
+      expect(roundtrip(map)).toStrictEqual(map);
+    });
+
+    it("should handle entries", () => {
+      map.set("hello", "world");
+      expect(roundtrip(map)).toStrictEqual(map);
+      map.set("world", "hello");
+      expect(roundtrip(map)).toStrictEqual(map);
+    });
+  });
+  describe("Set", () => {
+    beforeEach(() =>{
+      aSet = new Set<string>();
+    });
+
+    it("should handle an empty set", () => {
+      expect(roundtrip(aSet.values())).toStrictEqual([]);
+    });
+
+    it("should handle entries", () => {
+      aSet.add("hello");
+      expect(roundtrip(aSet).values()).toStrictEqual(aSet.values());
+      aSet.add("world");
+      expect(roundtrip(aSet).values()).toStrictEqual(aSet.values());
+    });
+  });
+});
+
+let map: Map<string, string>;
+let aSet: Set<string>;
```

Comment:
This all looks good, just some cosmetic questions

1. why the asymmetry in naming, `map` vs `aSet`?
2. why define them after they are used, rather than above the two `describe` blocks above?